### PR TITLE
fix(toAscii): add nukta combination mappings to single character

### LINKED ISSUE #146: Add Unicode Nukta Character combinations to toAscii
**Describe the bug**

Unicode allows two ways to type nukta characters. 

First is the defined code point, `ਸ਼` (U+0A36) which is a single character. The other method is to add a nukta char `਼` (U+0A3C) to an existing char, such as `ਸ਼` (U+0A38 + U+0A3C)

Since nutka is mapped to `æ`, the conversion results in `sæ` instead of `S`

**Expected behavior**

`ਸ਼` (U+0A38 + U+0A3C) => `S`
...and so on for other Pair Bindi chars

### PR DESCRIPTION
**Pull request by Sarabveer**
### Summary of PR
Adds mappings for `akhar + nukta` to single ascii character

### Linked issues
Fix #146

### CODE REVIEW

File: lib/toAscii.js [hunk 1]
```diff
@@ -27,7 +27,12 @@ const postReplacements = [
   [ /MØI/ug, 'µØI' ], // Use centered tippi on top of spacer char
   [ /([@R®H´ÍÏçœ˜†])u/ug, '$1ü' ], // Use lower aunkar when char is pair akhar
   [ /([@R®H´ÍÏçœ˜†])U/ug, '$1¨' ], // Use lower dulankaar when char is pair akhar
-  [ /(.)iÎ/ug, 'i$1Î' ], // Swap i with Î
+  [ 'sæ', 'S' ],
```
Comment (Harjot1Singh) [inline lib/toAscii.js:30]: Why are these post replacements and not mappings?

Comment (Sarabveer) [reply lib/toAscii.js:30]: They should be in `unicodeMappings.json`, however, it doesn't allow for multiple ASCII mappings.

For example, `S` would need to be mapped twice, but that is not allowed.

Comment (Harjot1Singh) [reply lib/toAscii.js:30]: Ok. To make it extra clear that they're not post-replacement rules, could you extract them into a separate mappings array and then spread it into the final list?

Comment (Sarabveer) [reply lib/toAscii.js:30]: Sure

**Pull request merged**